tests/lib.rs

```diff
@@ -196,10 +203,10 @@ fn test_object_group_property() {
     let mut cache = FilesystemResourceCache::new();
 
     let r = Map::parse_file("assets/tiled_object_groups.tmx", &mut cache).unwrap();
-    let prop_value: bool = if let Some(&PropertyValue::BoolValue(ref v)) = r
-        .layers()
-        .nth(1)
-        .unwrap()
+    let group_layer = r.get_layer(1).unwrap();
```

Comment:
Why'd this change? Does the object layer example have group layers?

Reply:
Yeah, it was using a group all along, but apparently this was just ignored by the XML parser before:

https://github.com/mapeditor/rs-tiled/blob/df98e9f35afb6d6a7d3b431537682ee67382fb68/assets/tiled_object_groups.tmx#L17-L23